python/cugraph/cugraph/datasets/metadata/dolphins.yaml

```diff
@@ -1,11 +1,13 @@
 name: dolphins
 file_type: .csv
+description: null
```

Comment:
Still need a description here. [Here's](http://www.casos.cs.cmu.edu/computational_tools/datasets/external/dolphins/index11.php) something I found for reference.